src/test/java/com/boomi/flow/services/boomi/mdh/DatabaseLoadGoldenRecordTests.java

```diff
@@ -332,6 +331,215 @@ public void testLoadWithComprehensiveFilter() {
                 );
     }
 
+    @Test
+    public void testLoadWithSimpleOrdering() {
+        List<String> uniqueIds = Arrays.asList(
+                "field 1 1",
+                "field 2 1",
+                "field 3 1",
+                "field 4 1",
+                "field 4 1 property",
+                "field 1 2",
+                "field 2 2",
+                "field 3 2",
+                "field 4 2",
+                "field 4 2 property"
+        );
+        List<String> names = Arrays.asList(
+                "field 1 1",
+                "field 2 1",
+                "field 3 1",
+                "field 4 1",
+                "field 4 1 property",
+                "field 1 2",
+                "field 2 2",
+                "field 3 2",
+                "field 4 2",
+                "field 4 2 property"
+        );
+
```

Comment:
could this scenario be simplified using less names/uniqueIds? They don't seem to add value to the test and it make it more complex to maintain and understand. The same apply to the rest of filter tests.

Reply:
It think I can, I'll reduce those to only a couple then.